docs/UserGuide.md

```diff
@@ -586,15 +596,16 @@ If you would like to delete a class, you can make use of this command.
 * Deletes the class at the specified `INDEX`.
 
 For example:
-* You have just received news that one of your classes, CS2100 Lab, has been permanently removed due to low enrollment.
-Hence, you want to delete the class from your Tutor's Pet.
-You type the command `delete-class 2` and press <kbd>Enter</kbd>.
+1. You have just received news that one of your classes, CS2100 Lab, has been permanently removed due to low enrollment.
+   Hence, you want to delete the class from your Tutor's Pet.
+
+1. Type the command `delete-class 2` and press <kbd>Enter</kbd>.
```

Comment:
Not sure if this would sound more user-centric(?) and more consistent with your other commands like add-class

```suggestion
1. You type `delete-class 2` and press <kbd>Enter</kbd>.
```

Reply:
same for the commands below...